### PR TITLE
 start tightening scheme usage in oc

### PR DESCRIPTION
**Pull request by deads2k**
This builds on previous scheme installation fixes and printing protections to start collapsing onto the external only scheme in kubectl for reading and printing.  It also fixes the upstream internally versioned commands which we inadvertently broke.  We're winning!

/assign @juanvallejo @soltysh @mfojtik

### CODE REVIEW

File: hack/lib/constants.sh [hunk 1]
```diff
@@ -341,8 +341,9 @@ function os::build::check_binaries() {
   # enforce that certain binaries don't accidentally grow too large
   # IMPORTANT: contact Clayton or another master team member before altering this code
   if [[ -f "${OS_OUTPUT_BINPATH}/${platform}/oc" ]]; then
-    if [[ "$(du -m "${OS_OUTPUT_BINPATH}/${platform}/oc" | cut -f 1)" -gt "115" ]]; then
-		  os::log::fatal "oc binary has grown substantially. You must have approval before bumping this limit."
+    ocsize=$(du -m "${OS_OUTPUT_BINPATH}/${platform}/oc" | cut -f 1)
+    if [[ "${ocsize}" -gt "116" ]]; then
```
Comment (mfojtik) [inline hack/lib/constants.sh]: +1MB ? :-))

File: pkg/api/legacygroupification/groupification.go [hunk 2]
```diff
@@ -31,6 +31,15 @@ import (
 	"github.com/openshift/origin/pkg/user/apis/user"
 )
 
+func IsOAPI(gvk schema.GroupVersionKind) bool {
```
Comment (mfojtik) [inline pkg/api/legacygroupification/groupification.go]: I would maybe put a DEPRECATED label here.

File: hack/lib/constants.sh (hunk 1 above)
Comment (deads2k) [reply hack/lib/constants.sh]: > +1MB ? :-))

Yeah, we were missing an api group.

File: pkg/api/legacygroupification/groupification.go (hunk 2 above)
Comment (deads2k) [reply pkg/api/legacygroupification/groupification.go]: > I would maybe put a DEPRECATED label here.

done

**Pull request merged**